f2/src/locales/en.json

```diff
@@ -135,10 +135,10 @@
   "howDidItStartPage.nextPage": "Next, you will be asked about what might be affected.",
   "howDidItStartPage.title": "How did the incident start?",
   "howDidTheyReachYou.app": "Through a program or an app",
-  "howDidTheyReachYou.email": "By email",
+  "suspect.email": "By email",
```

Comment:
I think that (at least for now) we don’t want to do this, as it goes against the way we’ve been choosing keys: by using the name of the page as the prefix. Otherwise the keys in the lingui file will no longer be next to the others on the same page (when the file is in alphabetical order) and could end up conflicting with other keys. I would suggest that for this PR we just rename suspectClues1, 2, and 3.

Reply:
Sounds good @sastels will follow the naming pattern suggested and update this shortly